package.json

```diff
@@ -9,6 +9,9 @@
   },
   "dependencies": {
     "@auth0/auth0-vue": "^1.0.3",
+    "@vuelidate/core": "^2.0.0",
```

Comment:
What are those new dependencies for?

Reply:
A library for validation, but it didn't end up being used, will be removed.